components/eda/code/eda-bbe/graphql.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'GraphQL subscription'
-description: Ballerina simplifies GraphQL subscriptions, allowing easy event streaming to frontend applications, enabling effortless event subscriptions and delivery to popular clients.
+description: Ballerina simplifies GraphQL subscriptions, allowing easy event streaming to frontend applications and also enabling effortless event subscriptions and delivery to popular clients.
```

Comment:
I prefer the one without `and also` : The sentence maintains its clarity and readability without the addition of "and also," making it a smoother read. @shafreenAnfar wdyt?

Reply:
Reverted to the original one.

Reply:
Yeah, I think the same.